wikipendium/wiki/templates/article.html

```diff
@@ -3,6 +3,21 @@
 
 {% block title %}{{ articleContent.get_full_title }} -{% endblock %}
 
+{% block meta %}
+<meta property=og:title content="{{ articleContent.get_full_title }}" />
+<meta property=og:type content=article />
+<meta property=og:image content=https://www.wikipendium.no/static/apple-touch/icon-114x114.png />
+<meta property=og:url content="{{ articleContent.get_absolute_url }}" />
```

Comment:
Sounds like `get_absolute_url` should return an absolute url rather than a relative url, then :P